doc/configuration.rst

```diff
@@ -171,6 +171,33 @@ Specifies the options to build the shapes in which the region of interest (``cou
    :widths: 25,10,22,27
    :file: configtables/build_shape_options.csv
 
+.. _subregion_cf:
+
+``subregion``
+=============================
+
+If enabled, this option allows a region of interest (``countries``) to be redefined into subregions,
+which can be activated at various stages of the workflow. Currently, it is only used in the ``simplify_network`` rule.
+
+.. literalinclude:: ../config.default.yaml
+   :language: yaml
+   :start-at: subregion:
+   :end-at: path_custom_shapes:
+
+.. csv-table::
+   :header-rows: 1
+   :widths: 25,10,22,27
+   :file: configtables/subregion.csv
+
+There are several formats for GADM IDs depending on the version, so before using this feature, please review the ``resources/shapes/gadm_shape.geojson`` file which can be created using the command:
+
+.. code:: bash
+
+    snakemake -j 1 build_shapes
+
+.. note::
+   The rule ``build_shapes`` currently use `Version 4.1  <https://geodata.ucdavis.edu/gadm/gadm4.1/gpkg/>`_ for their GADM data.
+
```

Comment:
Is it relevant for the model documentation? Geodata can change their conventions, and we also aim to transition to another dataset which a more permissive license.